ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2021.3.29f1
-m_EditorVersionWithRevision: 2021.3.29f1 (204d6dc9ae1c)
+m_EditorVersion: 2021.3.18f1
```

Comment:
You shouldn't be changing the Unity version for this project. It is likely that this screwed up internal files within the internal Unity management so I am going to ask you to move your work to a different branch after downloading the correct version and moving these files over.